components/CountUp.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+const CountUp = ({startTime}) => {
+	const [time, setTime] = React.useState(new Date(startTime).getTime());
+
+	React.useEffect(() => {
+		setInterval(() => {
+			setTime((time) => time + 1000);
+		}, 1000);
+	}, []);
```

Comment:
I think you missed the interval cleanup here. You can store the return of the interval in a ref and then call `clearInterval(intervalRef.current)` in the useEffect inner return, which is responsible to be called when the component is unmounted :)

Reply:
Hey @diegonvs I've done it in a different way, how can I confirm if it's good enough?

Reply:
You can place a breakpoint in the cleanup function and see if the intervalID(the interval variable in your fix) it's the same as registered before :)